app/graph/types/bot_user_type.rb

```diff
@@ -1,37 +1,35 @@
 BotUserType = GraphqlCrudOperations.define_default_type do
   name 'BotUser'
-  description 'Bot User type'
+  description 'A user type that represents a bot.'
 
   interfaces [NodeIdentification.interface]
 
-  field :avatar, types.String
-  field :name, types.String
-  field :get_description, types.String
-  field :get_version, types.String
-  field :get_source_code_url, types.String
-  field :get_role, types.String
-  field :identifier, types.String
-  field :login, types.String
-  field :dbid, types.Int
-  field :installed, types.Boolean
-  field :installations_count, types.Int
-  field :settings_ui_schema, types.String
-  field :installation, TeamBotInstallationType
-
-  field :settings_as_json_schema do
-    type types.String
-    argument :team_slug, types.String # Some settings options are team-specific
+  field :avatar, types.String, 'Picture' # TODO Merge with User.profile_image?
+  field :name, types.String, 'Name'
+  field :get_description, types.String, 'Description' # TODO Rename to 'description'
+  field :get_version, types.String, 'Version' # TODO Rename to 'version'
+  field :get_source_code_url, types.String, 'Source code URL' # TODO Rename to 'source_code_url'
+  field :get_role, types.String, 'Role of the bot when added to a team'
+  field :identifier, types.String, 'Machine name' # TODO Merge with BotUser.login?
+  field :login, types.String, 'Login name'
+  field :installed, types.Boolean, 'Is the bot approved for Check-wide use?' # TODO Rename to is_approved
```

Comment:
Actually this returns `true` if the bot is installed in `Team.current`... I think it should receive the team as an argument